compiler/src/iree/compiler/Modules/IO/Parameters/Conversion/StreamToParams/Patterns.cpp

```diff
@@ -42,8 +42,12 @@ struct ParameterLoadOpPattern
 
     // Lookup the appropriate device/queue for allocation based on the buffer
     // propreties.
+    auto memoryTypeOp =
```

Comment:
(same as in other file with using `Value memoryTypesValue = ` to avoid the op/getResult)